src/cosmograph/base.py

```
@@ -393,7 +392,7 @@ def none_are_none(self, names):
 # --------------------------------------------------------------------------------------
 # Some old code that might be useful
 
-from typing import List, Optional, Tuple
+from typing import List
```

Ruff (0.8.2)

Module level import not at top of file

(E402)